etl/recipes/etl.yaml

```diff
@@ -75,14 +79,14 @@ cooking:
               - bp-datapoints-translated-aligned
           options:
               op:
-                  coal_consumption_total: "coal_consumption_mtoe * 1000000"
+                  coal_consumption_total: "coal_consumption_exajoules"
                   electricity_generation_total: "electricity_generation * 1000000000"
-                  natural_gas_production_total: "gas_production_mtoe * 1000000"
-                  gas_proved_reserves_history: "gas_proved_reserves_history * 900000000"
-                  oil_consumption_total: "oil_consumption_tonnes * 1000000"
-                  oil_production_total: "oil_production_tonnes * 1000000"
-                  oil_proved_reserves_total: "oil_proved_reserves_history * 136400000"
-                  natural_gas_proved_reserves_total: "gas_proved_reserves_history * 900000000"
+                  natural_gas_production_total: "gas_production_exajoules"
+                  gas_proved_reserves: "gas_proved_reserves * 900000000"
```

Comment:
Is removing `_history` from this indicator on purpose?
Upon closer inspection it seems this `gas_proved_reserves` isn't used at all; it is filtered out in next procedure. Guess it can go?

Reply:
> Is removing _history from this indicator on purpose?
Yes, it was already done in the etl script of _ddf--bp--energy_ , so I updated it here accordingly.
_gas_proved_reserves_history --> gas_proved_reserves_ 

I'll remove _gas_proved_reserves_ , and put this info into commit message for the record.